docs/tomcat_install.md

```diff
@@ -2,12 +2,15 @@
 ## Getting Started
 ### Windows
 
+#### Prerequisites
 1. Install and Setup a supported Database:
     - [PostgreSQL (Tutorial)](./postgres_install.md)
     - [Microsoft SQL Server (Tutorial)](./mssql_install.md)
+2. Install Java 8: https://www.java.com/en/download/ (**Only this version is supported**! Please make sure that Tomcat uses Java 8 JDK)
+3. Install Tomcat 9: https://tomcat.apache.org/download-90.cgi (Download the 32-bit/64-bit Windows Service Installer).<br>
+   *You also have to set up a gui account in the installation!*
 
-2. Install Java 8: https://www.java.com/en/download/ (Only this version is supported! Please make sure that Tomcat uses Java 8 JDK)
-3. Install Tomcat 9: https://tomcat.apache.org/download-90.cgi (Download the 32-bit/64-bit Windows Service Installer, you also have to set up a gui account in the installation!)
+####Deploying Release
```

Comment:
```suggestion
#### Deploying Release
```

Perhapse try the suggestion above?
I still have the buggy formation.

![grafik](https://user-images.githubusercontent.com/81113086/122722960-a7e77280-d272-11eb-9dbe-cd201274e64f.png)